package.json

```diff
@@ -23,5 +23,9 @@
   "bugs": {
     "url": "https://github.com/digitalbazaar/bedrock-vue-address-form/issues"
   },
-  "homepage": "https://github.com/digitalbazaar/bedrock-vue-address-form"
+  "homepage": "https://github.com/digitalbazaar/bedrock-vue-address-form",
+  "devDependencies": {
+    "eslint-config-digitalbazaar": "^1.6.0",
+    "eslint-plugin-vue": "^5.2.2"
```

Comment:
I believe we are including `eslint` here as well.

Reply:
- [ ] include eslint